packages/dapp-svelte-wallet/react-ui/public/index.html

```diff
@@ -4,12 +4,22 @@
     <meta charset="utf-8" />
     <script src="lockdown.umd.js"></script>
     <script>
-      if ('%NODE_ENV%' === 'production') {
-        const { pow: mathPow } = Math;
-        Math.pow = (base, exp) => (typeof base === 'bigint' && typeof exp ==='bigint') ? base ** exp : mathPow(base, exp);
-        lockdown({ __allowUnsafeMonkeyPatching__: 'unsafe', errorTaming: 'unsafe', overrideTaming: 'severe' });
-        console.log('lockdown done.');
-      }
+      // Allow the React dev environment to extend the console for debugging
+      // features.
+      const consoleTaming = '%NODE_ENV%' === 'production' ? 'safe' : 'unsafe';
+      
+      const { pow: mathPow } = Math;
+      Math.pow = (base, exp) => 
+        (typeof base === 'bigint' && typeof exp ==='bigint') 
+        ? base ** exp : mathPow(base, exp);
+      
+      lockdown({ 
+        __allowUnsafeMonkeyPatching__: 'unsafe',
```

Comment:
Do we still need this? Please try removing and let me know what breaks.

Reply:
In prod, changing this  to 'safe' gives the following:
```
Uncaught TypeError: Cannot define property decorate, object is not extensible
  at Function.defineProperty (<anonymous>)
  at injectedScript.bundle.js:67
  at injectedScript.bundle.js:67
  at injectedScript.bundle.js:67
  at injectedScript.bundle.js:67
  at Object.<anonymous> (injectedScript.bundle.js:67)
  at Object.<anonymous> (injectedScript.bundle.js:67)
  at r (injectedScript.bundle.js:1)
  at Object.<anonymous> (injectedScript.bundle.js:25)
  at r (injectedScript.bundle.js:1)
  ```
In dev, the above occurs as well as:
```
Uncaught TypeError: Cannot assign to read only property 'Symbol(Symbol.iterator)' of object '[object Generator]'
  at index.js:1
  at Object.<anonymous> (index.js:1)
  at r (index.js:1)
  at Object.<anonymous> (index.js:1)
  at r (index.js:1)
  at Module.<anonymous> (index.js:1)
  at r (index.js:1)
  at index.js:1
  at index.js:1
  at index.js:1
  ```